queries.sql

```diff
@@ -0,0 +1,54 @@
+-- queries for the given questions
+--Question 1:
+SELECT sender_id, receiver_id, amount, created_at 
```

Comment:
Generate transaction history for a specific customer or account for a given date range.

Reply:
Okay sir